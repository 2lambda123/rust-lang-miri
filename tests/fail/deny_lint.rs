//@error-pattern: miri cannot be run on programs that fail compilation

#![deny(warnings)]

struct Foo;
//~^ ERROR: struct `Foo` is never constructed

fn main() {}
