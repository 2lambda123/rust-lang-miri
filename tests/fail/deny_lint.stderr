error: struct `Foo` is never constructed
  --> $DIR/deny_lint.rs:LL:CC
   |
LL | struct Foo;
   |        ^^^
   |
   = note: `-D dead-code` implied by `-D unused`

error: miri cannot be run on programs that fail compilation

error: aborting due to 2 previous errors

